topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.md

```diff
@@ -500,7 +500,7 @@ The remaining portion of this tutorial can also be viewed as a video:
 > 1. Run {% tool [Faster Download and Extract Reads in FASTQ](toolshed.g2.bx.psu.edu/repos/iuc/sra_tools/fasterq_dump/2.10.9+galaxy0) %} with the following parameters:
 >    - *"select input type"*: `List of SRA accession, one per line`
 >        - The parameter {% icon param-file %} *"sra accession list"* should point the output of the {% icon tool %} "**Cut**" from the previous step.
->    - **Click** the `Execute` button. This will run the tool, which retrieves the sequence read datasets for the runs that were listed in the `SRA` dataset. It may take some time. So this may be a good time to do get coffee.
+>    - **Click** the `Execute` button. This will run the tool, which retrieves the sequence read datasets for the runs that were listed in the `SRA` dataset. It may take some time. So this may be a good time to go get coffee.
```

Comment:
```suggestion
>    - **Click** the `Execute` button. This will run the tool, which retrieves the sequence read datasets for the runs that were listed in the `SRA` dataset. It may take some time. So this may be a good time to take a break.
```

Reply:
I don't think there's anything wrong with "go get coffee" - it's the author's style, I would leave it as is.

Reply:
Maybe I've been living in UK for too long, but I think "take a break" saves from debates about what's the best drink between tea and coffee :D

Reply:
Indeed it does. That is a fine point I haven't considered. Now I feel compelled to accept the suggestion :smile: